packages/apollo-server-types/src/index.ts

```diff
@@ -79,6 +79,12 @@ export interface ApolloConfig {
   serverlessFramework: boolean;
 }
 
+export interface ServeHtmlOptions {
```

Comment:
Curious: Using `ServeHtmlOptions` here versus `Serve`**`HTML`**`Options` to align with `HTML` capitalization/usage in other places because... this is public-facing?  (I'm a fan of not capitalizing initialisms in camelCase, as you may know).  Just trying to understand.

Reply:
I'm not sure where we use capitalized HTML? But in any case, oops, this type was a leftover from an earlier version of this PR and should be removed.

Reply:
Ah I guess I wrote some internal `prefersHTML` functions. I'll rename them.